util/git/client.go

```diff
@@ -421,10 +421,16 @@ func (m *nativeGitClient) Fetch(revision string) error {
 func (m *nativeGitClient) LsFiles(path string, enableNewGitFileGlobbing bool) ([]string, error) {
 	if enableNewGitFileGlobbing {
 		// This is the new way with safer globbing
-		err := os.Chdir(m.root)
+		cwd := m.root
```

Comment:
Just for my understanding, this was just added for debugging and doesn't make any material change?